locations/kub.yml

```diff
@@ -89,10 +39,34 @@ networks:
     role: mesh
     name: mesh_simeon
     prefix: 10.31.139.16/32
-    ipv6_subprefix: -1
+    ipv6_subprefix: -10
+
+ # MESH - 5 GHz 802.11s
+  - vid: 20
+    role: mesh
+    name: mesh_5g_ap1
+    prefix: 10.31.139.17/32
+    ipv6_subprefix: -20
+    mesh_ap: kub-ap1
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+
+  # MESH - 2.4 GHz 802.11s
+  - vid: 21
+    role: mesh
+    name: mesh_2g_ap1
+    prefix: 10.31.139.18/32
+    ipv6_subprefix: -21
+    # make mesh_metric(s) for 2GHz worse than 5GHz
+    mesh_metric: 1024
```

Comment:
this line should be removed as we now prefer 5 GHz over 2.4 GHz for 802.11s by default